cmd/apiserver/app/server/run_server.go

```diff
@@ -49,8 +49,10 @@ func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	glog.Infoln("Installing TPR types to the cluster")
 	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
 	if err := tprInstaller.InstallTypes(); err != nil {
-		glog.V(4).Infof("Installing TPR types failed, continuing anyway (%s)", err)
+		glog.Errorf("Failed to installing TPR types (%s)", err)
+		return err
```

Comment:
@simonleung8 I just want to make sure I understand here - when you return the error here, is stop chan closed by the caller of `RunServer`?